getting-started-with-spring-boot-on-aws/src/main/resources/application.yml

```diff
@@ -16,3 +16,7 @@ cloud:
       enabled: true
     sqs:
       enabled: true
+
+custom:
+  bucket-name: stratospheric-demo-application
```

Comment:
Are the changes for the `getting-started-with-spring-boot-on-aws` folder all strictly necessary? I'm asking because we would then differ from the blog post and changing it is a slow and cumbersome process. 

If they are minor cosmetic changes, I would opt in to leave it as is to keep the sync with the blog post.

Reply:
While technically not necessary, the `getting-started-with-spring-boot-on-aws` folder would be out-of-sync and inconsistent with the main application. Furthermore, it'd still be on Spring Boot 2.7.x in that case, which we'd mean we'd have to maintain two versions at this point.